robot/Cumulus/tests/browser/gift_entry/error_validation_rename_fields.robot

```diff
@@ -70,11 +70,12 @@ Validate Errors When Field Is Renamed
     Set Suite Variable                  ${BATCH1_Id}
     Page Should Not Contain Locator     gift_entry.page_error
     Page Should Not Contain Locator     label                         Account 1: custom_acc_text
+    Scroll Page To Location             10     10
```

Comment:
this is not a good practice. Instead please use Scroll Element Into View and provide the locator for one of the elements on the page which you would like to scroll using locator manager strategy

Reply:
Fixed this by adding a locator.